storybook/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "scripts": {
     "build": "storybook build --output-dir dist/ --config-dir config/ --quiet && touch dist/.nojekyll",
+    "build:chromatic": "IS_CHROMATIC=true storybook build --output-dir dist/ --config-dir config/ --quiet && touch dist/.nojekyll",
```

Comment:
Can we reuse the `build` script here? So something like `"build:chromatic": "IS_CHROMATIC=true npm run build"`?

This would prevent the 2 scripts from drifting apart over time.